compat/hwc2/hwc2_compatibility_layer.cpp

```diff
@@ -132,6 +132,12 @@ hwc2_compat_display_t* hwc2_compat_device_get_display_by_id(
     return display;
 }
 
+void hwc2_compat_device_destroy_display(hwc2_compat_device_t* device,
```

Comment:
Would it make more sense to have hwc2_compat_device_destroy_display(hwc2_compat_device_t* device, hwc2_compat_display_t* display) so it also frees the compat struct and is in line with how hwc2_compat_display_destroy_layer works?

Reply:
It does indeed, and should be addressed. Thanks!